pkg/apiserver/webhooks/validating-webhook/validating-webhook.go

```diff
@@ -196,6 +196,21 @@ func admitDVs(ar *v1beta1.AdmissionReview) *v1beta1.AdmissionResponse {
 		return toAdmissionResponseError(err)
 	}
 
+	client := GetClient()
+	if client != nil {
+		_, err := client.CoreV1().PersistentVolumeClaims(dv.GetNamespace()).Get(dv.GetName(), metav1.GetOptions{})
+		if err == nil {
```

Comment:
Is it possible that PVC does not exist and still err=nil since there was no explicit error in CoreV1().PersistentVolumeClaims....what happens if this changes in the future?
I recommend doing 'pvc, err' := client.CoreV1().PersistentVolumeClaims(dv.GetNamespace()).Get(dv.GetName(), metav1.GetOptions{})
and then check that pvc is not nil. This way, if  tomorrow  CoreV1().PersistentVolumeClaims is changed and the logic is - if there is no pvc, returns nil,nil (and not nil,err) since there was not error, and not because it could not find pvc, you are good.

Reply:
OK, it seems like pvc is not `nil` if it doesn't exist and we get an empty object.
I changed it to get list of pvcs in the same namespace and fail if I find pvc with the same destination name.

Reply:
+1. just a little question....